reinforcement_learning/bindings/python/README.md

```diff
@@ -1 +1,14 @@
+# Python Bindings
+
+## Build + Install
+1. [Install pip for python3.6](https://askubuntu.com/questions/889535/how-to-install-pip-for-python-3-6-on-ubuntu-16-10)
+2. `sudo python3.6 -m pip install wheel`
```

Comment:
Can we add a "requirements.txt" and replace this with "pip install -r requirements.txt"? This will prevent the need to update this document if the dependencies change.